continuous_integration/travis/install.sh

```diff
@@ -73,7 +73,7 @@ pip install --upgrade --no-deps git+https://github.com/dask/zict
 pip install --upgrade --no-deps git+https://github.com/dask/distributed
 pip install --upgrade --no-deps git+https://github.com/dask/s3fs
 
-if [[ $PYTHONOPTIMIZE != '2' ]] && [[ $NUMPY > '1.11.0' ]] && [[ $NUMPY < '1.14.0' ]]; then
+if [[ $PYTHONOPTIMIZE != '2' ]] && [[ $NUMPY > '1.11.0' ]] && [[ $NUMPY < '1.16.5' ]]; then
```

Comment:
cc @martindurant it looks like we aren't installing fastparquet or pyarrow in CI, except for old numpy versions

Reply:
Definitely seen lots of parquet things show up in the past, so that would be quite a surprise. That line from https://github.com/dask/dask/pull/3088 ; pyarrow is included in the dev installs, later in the file.

Reply:
@martindurant I noticed the test case I added wasn't running in CI, along with the other parquet tests, until I made this change.